### PR TITLE
Add ability to make concepts public.

### PR DESCRIPTION
**Pull request by nsthorat**
Also refactors concepts a little bit so that there's a separated metadata section for concepts.

image

https://huggingface.co/spaces/lilacai/nikhil_staging

Public nikhil concept:
https://lilacai-nikhil-staging.hf.space/concepts#100712716653593140239/aliens
Private nikhil concept (this should error):
https://lilacai-nikhil-staging.hf.space/concepts#100712716653593140239/aliens_private

image

### CODE REVIEW

File: web/blueprint/src/lib/components/commands/CommandCreateConcept.svelte
Comment (dsmilkov) [inline web/blueprint/src/lib/components/commands/CommandCreateConcept.svelte:1]: Small typo in the bolded enable -> disable

...Authentication is enabled, so LLM generation of examples is disabled. Please fork this and **enable** auth..."

File: web/blueprint/src/lib/components/concepts/ConceptView.svelte
Comment (dsmilkov) [inline web/blueprint/src/lib/components/concepts/ConceptView.svelte:1]: where we show this concept is publicly visible, can we add a small button that copies the url in paste memory

**Review by dsmilkov — APPROVED** [review]
it's coming along!

File: web/blueprint/src/lib/components/commands/CommandCreateConcept.svelte
Comment (nsthorat) [reply web/blueprint/src/lib/components/commands/CommandCreateConcept.svelte:1]: Done

File: web/blueprint/src/lib/components/concepts/ConceptView.svelte
Comment (nsthorat) [reply web/blueprint/src/lib/components/concepts/ConceptView.svelte:1]: Done

**Pull request merged**